.github/workflows/e2e-parallel-destroy.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       - name: Setup Terraform
         uses: hashicorp/setup-terraform@v3
         with:
-          terraform_version: 1.0.0
+          terraform_version: 1.3.10
```

Comment:
The EKS module now requires MSTV of 1.3 so updating this in our e2e tests as well